src/client/components/RobotLogo/RobotLogo.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Robot from '../../assets/images/robot_logo.png';
+import Arrow from '../../assets/images/arrow.png';
+import './RobotLogo.css';
+
+export default function RobotLogo() {
+  return (
+    <div className="RobotLogo">
+      <div className="RobotLogoDiv">
```

Comment:
I think this should be a button, rather than a div. It can still contain an img and text. And if it is a button it should have an onClick, something like this as an attribute -       <button type="button" onClick={onClick}>
        Click me
      </button>

Reply:
Hi Beth
yea added a button as you suggested